### DISCUSSION TITLE
原来这就是绳网吗，{骂人死全家}

### DISCUSSION THREAD

**Discussion by ghost** [share121/inter-knot#5437] [绝区零]
原来就是github的套皮
hhh

**Comment by wshtdgo** [comment]
有点迷惑的言论哈

**Comment by Loxia1121** [comment]
套皮有什么问题吗？

**Comment by TNCNET** [comment]
是，
但是如果你有办法做的更好，并且已经实践成功了，那么你可以这么评价。否则不要这么说。

**Comment by WinterSamu** [comment]
你是不是有什么毛病？按你这么说是不是要用服务器了呗，你有能耐你整去，在那装尼玛呢。有的是这种套皮论坛，有本事在 GitHub 上都发一边！不要碧莲的东西。滚！

**Reply by ghost** [reply]
你妈了个逼的，你个孤儿，我操你妈的，你要是牛逼，你不怕米哈游的法务部，你他妈给我做一个。妈了个逼的，你又不敢做，你让你做你也做不了啊，这不告诉你也做不了啊，你吹你妈牛逼呢你。套皮你敢做吗？你做的出来吗？妈了个逼的，死全家的玩意儿，你全家也都死全家。你全家的全家死全家。你老师死全家，你朋友死全家，你妈你爹全死全家。

**Reply by WinterSamu** [reply]
您与其在这复制粘贴，还不如自己动手。笑死。
宁赢了，我骂不过你，差不多得了😅

**Reply by WinterSamu** [reply]
只能说您活该了，大家都清楚的事用宁碍什么眼。显得自己撅醒了吗？笑得我都不行啦🤣
世界网站千千万，哪里没有你的存身之处？有本事去当绳网润人去，别在这里丢人现眼，满意就待不满意就滚。蒸鹅心馁~

**Reply by WinterSamu** [reply]
呦，骂完人后就注销，真是玩不起啊。😂
反正骂完人后，精神胜利！甭管怎么滴，先赢了再说，赢！注销账号后，再也看不到反驳的声音，赢赢赢！这就是阿Q的皮套吗？爱了爱了。笑得我都不行啦🤣
哎呦，可惜咱们绳网庙小容不下你这大佛啊，建议润吧赶紧润，当个绳网润人二游润人也不错。在这里的，谁不清楚这个事？从登录 Github 这一刻开始。这又是撅醒了吗？在这里显得你有哪门子优越感？笑死。爱待待不待滚
不知道这位仁兄会不会在看捏？看完之后再来次精神胜利呢？这就不得而知喽。
